packages/jw-player/src/mixins/chapter/index.css.js

```diff
@@ -0,0 +1,59 @@
+import { css } from 'lit';
+
+export const style = css`
```

Comment:
@anoblet same here, maybe create `scss`, generate css like how `cxl-ui` does it?